build/common.sh

```diff
@@ -737,27 +739,36 @@ function kube::release::sha1() {
 function kube::release::create_docker_images_for_server() {
```

Comment:
While you're at it, document the args this function takes.  (See `kube::release::gcs::publish_ci`.)